manifests/base/repo-server/argocd-repo-server-deployment.yaml

```diff
@@ -23,8 +23,6 @@ spec:
         imagePullPolicy: Always
         args:
           - /usr/local/bin/argocd-repo-server
-          - "--redis"
-          - "$(ARGOCD_REDIS_SERVICE):6379"
```

Comment:
This will now be provided via `REDIS_SERVER`. No need to set it in non-HA, because we already default to `argocd-redis:6379`.

Reply:
where is the default defined?

Reply:
The default is (for some reason) applied here: https://github.com/argoproj/argo-cd/blob/cac553ab50344c45de4f10d5e5a1aed079b94e6e/util/cache/cache.go#L142